README.md

```diff
@@ -33,6 +35,23 @@ game titles.
 
 At the end, the installation script prints the URL where you can access the demo. Happy gaming!
 
+## Uninstall
+
+In case that you don't want to use the cloud gaming demo any longer you can simply uninstall it from
+your Anbox Cloud Appliance by running the following commands
+
+    cd cloud-gaming-demo
+    sudo ./uninstall.sh
+
+Afterwards all configuration and the cloud gaming demo itself is removed from the machine.
+
+## Used games
+
+The demo uses the following games:
+
+ * [BombSquad](http://www.bombsquadgame.com/)
+ * [Mindustry](https://github.com/Anuken/Mindustry)
```

Comment:
As we don't use the game `Minetest`, could you
1. drop the `minetest` from the `GAMES` array in the `scripts/install.sh`.
2. update the screenshot to honor this change as well.

Reply:
1. is already done in https://github.com/anbox-cloud/cloud-gaming-demo/pull/12  and 2. I would like to keep as is

Reply:
Make sense to me.